tests/integration/test_io.py

```diff
@@ -273,8 +265,8 @@ def test_append(self):
 
         with NWBHDF5IO(self.path, mode='a') as io:
             nwb = io.read()
-            elec = nwb.processing['test_proc_mod']['LFP'].electrical_series['test_device'].electrodes
-            ts2 = ElectricalSeries(name='timeseries2', data=[4., 5., 6.], rate=1.0, electrodes=elec)
+            link_electrodes = nwb.processing['test_proc_mod']['LFP'].electrical_series['test_es'].electrodes
+            ts2 = ElectricalSeries(name='timeseries2', data=[4., 5., 6.], rate=1.0, electrodes=link_electrodes)
             nwb.add_acquisition(ts2)
             io.write(nwb)
 
```

Comment:
Can you add a line that checks (after reading, in the context below) that the two electrodes are the same object?  Something like:
```
self.assertIs(nwb.processing['test_proc_mod']['LFP'].electrical_series['test_es'].electrodes, 
              nwb.acquisition['timeseries2'].electrodes)
```

It would be nice to make sure the linked objects get read back in correctly.